kolibri/plugins/coach/assets/src/views/common/resourceSelection/subPages/PreviewSelectedResources/PreviewMetadata.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+
+  <HeaderTable
+    v-if="content"
+    class="license-detail-style"
+  >
+    <HeaderTableRow
+      v-if="content.duration"
+      :keyText="coreString('suggestedTime')"
+    >
+      <template #value>
+        {{ content.duration ? getTime(content.duration) : notAvailableLabel$() }}
```

Comment:
If this node is not rendered if !content.duration. We shouldnt take care of this ternary here?